### PR TITLE
Add VM OS and IP Inventory

### PR DESCRIPTION
**Pull request by Lainow**
*Please delete options that are not relevant.*

- [x] I have read the CONTRIBUTING document.
- [x] I have performed a self-review of my code.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] This change requires a documentation update.

## Description

- It fixes !34616 and !32800
- Here is a brief description of what this PR does

Addition of the operating system and ip in the information retrieved from the vCenter API by the agent

### CODE REVIEW

**Comment by stonebuzz** [comment]
@Lainow 

can you add test for OS (Even if the implementation was already OK on the inventory side)

File: phpunit/functional/Glpi/Inventory/Assets/VirtualMachineTest.php [hunk 1]
```diff
@@ -270,6 +276,20 @@ public function testImportVirtualMachine()
               'uuid' => '420904fe-6a92-95e8-13f9-a37fc3607c14',
           ]));
 
+          //get NetworkPort Computer
+          $netport_computer_linked_first = new \NetworkPort();
+          $this->assertTrue($netport_computer_linked_first->getFromDBByCrit([
+              'items_id' => $computer_linked_first->fields['id'],
+              'itemtype' => 'Computer',
+          ]));
```
Comment (stonebuzz) [inline phpunit/functional/Glpi/Inventory/Assets/VirtualMachineTest.php:284]: can you check IPAddress value

File: phpunit/functional/Glpi/Inventory/Assets/VirtualMachineTest.php [hunk 2]
```diff
@@ -270,6 +276,20 @@ public function testImportVirtualMachine()
...
+
+          //get Operating system Computer
+          $os_computer_linked_first = new \Item_OperatingSystem();
+          $this->assertTrue($os_computer_linked_first->getFromDBByCrit([
+              'items_id' => $computer_linked_first->fields['id'],
+              'itemtype' => 'Computer',
+          ]));
```
Comment (stonebuzz) [inline phpunit/functional/Glpi/Inventory/Assets/VirtualMachineTest.php:291]: can you check OS value

**Review by stonebuzz — CHANGES_REQUESTED** [review]

**Review by stonebuzz — APPROVED** [review]

**Comment by cedric-anne** [comment]
@Lainow 

Please rebase your PR to ensure that test suite is executed on an up-to-date branch.

**Pull request merged**